android/app/build.gradle

```diff
@@ -64,4 +64,6 @@ dependencies {
     testImplementation 'junit:junit:4.12'
     androidTestImplementation 'com.android.support.test:runner:1.0.2'
     androidTestImplementation 'com.android.support.test.espresso:espresso-core:3.0.2'
+    implementation 'com.google.firebase:firebase-analytics:17.2.0'
```

Comment:
As far as I know,  you can delete this line. This dependency should be inherently compiled (unless you know something otherwise)

Reply:
Sounds good! I was just following a tutorial and they added it. Will comment it out for now!